pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,22 +63,28 @@ type OpenShiftLogsOptions struct {
 }
 
 // NewCmdLogs creates a new logs command that supports OpenShift resources.
-func NewCmdLogs(name, parent string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+func NewCmdLogs(name, baseName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	o := OpenShiftLogsOptions{
 		KubeLogOptions: &kcmd.LogsOptions{},
 	}
+
 	cmd := kcmd.NewCmdLogs(f.Factory, out)
 	cmd.Short = "Print the logs for a resource."
 	cmd.Long = logsLong
-	cmd.Example = fmt.Sprintf(logsExample, parent+" "+name)
+	cmd.Example = fmt.Sprintf(logsExample, baseName, name)
 	cmd.SuggestFor = []string{"builds", "deployments"}
 	cmd.Run = func(cmd *cobra.Command, args []string) {
-		kcmdutil.CheckErr(o.Complete(f, out, cmd, args))
+		err := o.Complete(f, cmd, args, out)
+		kcmdutil.CheckErr(err)
```

Comment:
@bparees changed and squashed :)
I did so because either OpenShift or Kubernetes has implemented the two ways. In the next refactors + unittests I will change all to keep this pattern...